cfep-18.md

```diff
@@ -0,0 +1,79 @@
+
+## Rationale
+
+The current stack used to manage and consume conda-forge is tuned to deal with dynamic linkage between packages.
+As long as the solver can find a solution, updating a package version is simply updating the package itself, all other packages can stay untouched in the installations as long as they are compatible with the new version.
+If we would use static instead of shared linkage, we would need to rebuild the packages depending on that library.
+
+Rebuilding packages that depend on a library in the shared linkage setting are limited to the direct dependencies if there was an ABI breakage.
+If there was no ABI breakage, there is also no need to rebuild any downstream package.
+Users can simply update to the new version.
+
+Rebuilding packages with static linkage is more complicated as the symbols of the library are copied into every binary that uses the library.
+Thus there we would not only need to rebuild the packages that directly on the library but all the whole set of downstream packages that also have an indirect linkage as they also may contain copies of the symbols.
+As the symbols are copied, we would not even need to do these rebuilds for ABI / version changes but for every new build of the packaged library.
+
+As an additional complication, you also need to include the license of the library of the library in every package that includes symbols of a static library if the license of the library requires it.
+This means that executables that are statically linked also need to include the license headers of most of their libraries.
+
+Using dynamic linkage also has the benefit of a smaller disk usage even when the static archives are not installed.
+As we will have the symbols of a library copied into every binary that uses a library.
+In the dynamic setting, the symbols would only exist once on disk, inside the shared object.
+
+Not only does dynamic linkage reduce the storage requirements on disk but it also reduces the memory usage when several processes use the same library.
+Depending on the operation system, the symbols are then only loaded once into memory and shared between the processes.
+
+Build static archives (in separated packages) is still useful for some non-common workflows.
+These don't represent the majority of uses of our packages but can profit a lot from our using our existing infrastructure:
+
+* Building static binaries that can be shipped as a single executable to ease installation, e.g. `micromamba`
+* Building Python wheels for Windows: Wheels that include binary code link all their static libraries into their Python module and can profit from conda-forge having static builds available (note that this also requires the wheels or the user to separate supply the VC runtime conda-forge uses).
+
+* static binaries for linux that are shipped as a single file
+* windows python wheels
```

Comment:
These two lines seem like they were expanded into the lengthier list items? Looks like they can be deleted?